cmd/kubeadm/app/master/manifests.go

```diff
@@ -296,6 +299,18 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 		"--allow-privileged",
 	)
 
+	if cfg.AuthorizationMode != "" {
+		command = append(command, "--authorization-mode="+cfg.AuthorizationMode)
+		switch cfg.AuthorizationMode {
+		case authorizer.ModeABAC:
+			command = append(command, "--authorization-policy-file="+path.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, authorizationPolicyFile))
+		case authorizer.ModeRBAC:
+			command = append(command, "--runtime-config=rbac.authorization.k8s.io/v1alpha1")
```

Comment:
TODO to remove once https://github.com/kubernetes/kubernetes/pull/39625 merges, which will enable v1beta1 by default.